README.md

```diff
@@ -176,6 +176,12 @@ this shouldn't need to be performed manually.
 
 See the documentation on testing Cloe for more details.
 
+### Disabling Unit Tests
+
+When using `make` to build the project, add this to the command line:
```

Comment:
How about adding a sentence on how to do it with native conan? Not that there's much of a difference anyways, but still.